coreference_resolution/coref_sampling.py

```diff
@@ -0,0 +1,35 @@
+import json
+from stanfordcorenlp import StanfordCoreNLP
+
+def convert_chains (coref_chains):
+    new_coref_chains = list()
+    for chain in coref_chains:
+        corefs = list()
+        for ref in chain:
+            corefs.append(ref[3])
+        new_coref_chains.append(corefs)
+    return new_coref_chains
+
+if __name__ == '__main__':
+    f = open('../data/random_sample/sample.json')
+    data = json.load(f)
+    f.close()
+
+    # RUN THIS FIRST FROM NLP FOLDER: java -mx4g -cp "*" edu.stanford.nlp.pipeline.StanfordCoreNLPServer -port 9001 -timeout 15000 
+    nlp = StanfordCoreNLP('http://localhost', port=9001)
+
+    for i in data:
+        messages = ""
+        for j in i["messages"]:
+            messages += j["utterance"]
+            messages += ". "
```

Comment:
beskederne indeholder "." i forvejen ved nogle af sætningerne, så det vil stå der to gange i visse tilfælde